content/program-security.md

```diff
@@ -0,0 +1,681 @@
+# Create a Basic Program, Part 3 - Basic Security and Validation
+
+# Lesson Objectives
+
+_By the end of this lesson, you will be able to:_
```

Comment:
@ZYJLiu looks like this got changed to underscores in the last PR... would you mind changing it back to asterisks?  Thanks!